mscore/updatechecker.cpp

```diff
@@ -146,18 +146,16 @@ void UpdateChecker::check(QString currentVersion, bool m)
 #if defined(Q_OS_MAC)
       os = "mac";
 #endif
-      if (qApp->applicationName() == "MuseScore3") { //avoid nightly cymbals
-            if (MuseScore::unstable())
-                  release = "pre";
-            else
-                  release = "stable";
-            }
-      else {
+      if (MuseScore::unstable())
             release = "nightly";
-            }
+      else
+            release = "stable";
+
       if (MScore::debugMode)
             qDebug("release type: %s", release.toLatin1().constData());
-      if (!os.isEmpty() && !release.isEmpty() && release != "nightly") {
+      if (!os.isEmpty() && !release.isEmpty()) {
+             if (release == "nightly")
+                   _currentVersion = QString("%1.%2").arg(_currentVersion).arg(BUILD_NUMBER);
             _currentVersion =  currentVersion;
```

Comment:
I don't get lines 157 to 159. Why do you set `_currentVersion` to `QString("%1.%2").arg(_currentVersion).arg(BUILD_NUMBER)`, if it will anyways be set the line after to `currentVersion?`

Reply:
Good catch... these lines should go further down...